site/documentation/content/api/credentials/index.md

```diff
@@ -155,6 +155,7 @@ Below is an example for a payload containing [a hashed password]({{< relref "#ha
   "auth-id": "sensor1",
   "enabled": true,
   "secrets": [{
+    "id": "349556ea-4902-47c7-beb0-1009ab693fb4",
```

Comment:
IMHO there is no need to include the id in the response payload. What should a protocol adapter do with it?

Reply:
Indeed, the adapters don't have any use of this. I made a mistake by editing that file ! I reverted the changes